docs/installation/linux.md

```diff
@@ -83,6 +85,46 @@ or install it from the terminal:
 sudo installer -pkg powershell-6.0.0-alpha.9.pkg -target /
 ```
 
+Also install [Homebrew's OpenSSL][openssl]:
+
+```
+brew install openssl
+```
+
+[Homebrew][brew] is the missing package manager for macOS.
+If the `brew` command was not found,
+you need to install Homebrew following [their instructions][brew].
+
+.NET Core requires Homebrew's OpenSSL because the "OpenSSL" system libraries on macOS are not OpenSSL,
+as Apple deprecated OpenSSL in favor of their own libraries.
+This requirement is not a hard requirement for all of PowerShell;
+however, most networking functions (such as `Invoke-WebRequest`)
+do require OpenSSL to work properly.
+
+*Please ignore* .NET Core's installation instructions to manually link the OpenSSL libraries.
+This is *not* required for PowerShell as we patch .NET Core's cryptography libraries to find Homebrew's OpenSSL in its installed location.
+Again, *do not* run `brew link --force` nor `ln -s` for OpenSSL, regardless of other instructions.
```

Comment:
Can you check FAQ and other docs. I added force linking as a fix and dice enter it somewhere in our repo.